src/translator/CMakeLists.txt

```diff
@@ -13,10 +13,15 @@ add_library(bergamot-translator STATIC
     sentence_ranges.cpp
     service.cpp
 )
+if (USE_WASM_COMPATIBLE_MARIAN)
+  # Using wasm compatible marian should include this compile definition;
+  # Has to be done here because of the marian's cmake practices
+  target_compile_definitions(bergamot-translator PUBLIC WASM)
+endif()
 
 if(COMPILE_WASM)
   # A dirty hack because of marian's bad cmake practices
-  target_compile_definitions(bergamot-translator PUBLIC USE_SSE2 WASM)
+  target_compile_definitions(bergamot-translator PUBLIC USE_SSE2)
```

Comment:
I'm confused, what happens if one doesn't set -DUSE_WASM_COMPATIBLE_MARIAN and does set -DCOMPILE_WASM?